charts/incubator/broadcast-box/Chart.yaml

```
@@ -0,0 +1,37 @@
+annotations:
+  max_scale_version: 24.04.1
+  min_scale_version: 24.04.0
+  truecharts.org/category: media
+  truecharts.org/max_helm_version: "3.15"
+  truecharts.org/min_helm_version: "3.11"
+  truecharts.org/train: incubator
+apiVersion: v2
+appVersion: latest
+dependencies:
+  - name: common
+    version: 25.1.7
+    repository: oci://tccr.io/truecharts
+    condition: ""
+    alias: ""
+    tags: []
+    import-values: []
+deprecated: false
+description: Easily broadcast/stream to others with sub-second latency
+home: https://truecharts.org/charts/incubator/broadcast-box
+icon: https://truecharts.org/img/hotlink-ok/chart-icons/broadcast-box.webp
+keywords:
+  - broadcast-box
+  - broadcast
+  - stream
+kubeVersion: '>=1.24.0-0'
+maintainers:
+  - name: TrueCharts
+    email: info@truecharts.org
+    url: https://truecharts.org
+name: broadcast-box
+sources:
+  - https://github.com/Glimesh/broadcast-box
+  - https://hub.docker.com/r/seaduboi/broadcast-box
+  - https://github.com/truecharts/charts/tree/master/charts/incubator/broadcast-box
+type: application
+version: 0.1.0
```

```suggestion
version: 0.0.1
```
Initial version is this